etcdserver/etcdhttp/client_auth_test.go

```diff
@@ -22,10 +23,26 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/coreos/etcd/Godeps/_workspace/src/golang.org/x/crypto/bcrypt"
 	"github.com/coreos/etcd/etcdserver/auth"
 )
 
-const goodPassword = "$2a$10$VYdJecHfm6WNodzv8XhmYeIG4n2SsQefdo5V2t6xIq/aWDHNqSUQW"
+const goodPassword = "good"
+
+func setFakeHashFunctions() {
```

Comment:
The functions are used to generate hashed password from plaintext one, or compare hashed password with plaintext to see whether it is hashed by the plaintext one. I use HashFunctions because they are all related to hashing.